Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -49,26 +42,25 @@ open class ChatChannelListItemView<ExtraData: ExtraDataTypes>: ChatSwipeableList
     
     public required init(
         channel: _ChatChannel<ExtraData>? = nil,
-        userId: UserId? = nil,
-        uiConfig: UIConfig<ExtraData> = .default
+        userId: UserId? = nil
     ) {
         channelAndUserId = (channel, userId)
-        self.uiConfig = uiConfig
-        
         super.init(frame: .zero)
-        
-        self.uiConfig = uiConfig
     }
```

Comment:
I think you should remove the `init` completely and change `(channel: _ChatChannel<ExtraData>?, currentUserId: UserId?)` -> `(channel: _ChatChannel<ExtraData>, currentUserId: UserId)?`

Reply:
No need to set the tuple to be optional, the tuple default value is `(nil, nil)` so it works out of box. Anyway deleted the `init`, should be ready now :)